src/clientmetricreport/StreamMetricReport.ts

```diff
@@ -10,4 +10,5 @@ export default class StreamMetricReport {
   direction: Direction;
   previousMetrics: { [id: string]: number } = {};
   currentMetrics: { [id: string]: number } = {};
+  stringValues: { [id: string]: string } = {};
```

Comment:
same

Reply:
Updated it to `currentStringMetrics`. Also added comments that this is a workaround in case metric values are strings.